docs/configuration.asciidoc

```diff
@@ -389,6 +389,18 @@ Use this option to disable any of these.
 
 Get an array of enabled spies with `ElasticAPM.agent.config.enabled_spies`.
 
+[float]
+[[config-instrumented-rake-tasks]]
+==== `instrumented_rake_tasks`
+
+[options="header"]
+|============
+| Environment                           | `Config` key              | Default
+| `ELASTIC_APM_INSTRUMENTED_RAKE_TASKS` | `instrumented_rake_tasks` | `['app:custom:task', 'other_task']`
```

Comment:
Isn't the default value an empty array?

Reply:
You are right! I think I confused it with an example column.